at_client/util/encryptionutil.py

```diff
@@ -87,3 +89,33 @@ def private_key_from_base64(s):
     def public_key_from_base64(s):
         key_bytes = base64.b64decode(s.encode('utf-8'))
         return load_der_public_key(key_bytes)
+    
+    @staticmethod
+    def generate_iv_nonce():
+        return IVNonce().token_bytes()
+
+class IVNonce:
```

Comment:
Make sure random is seeded, I'm not sure what the recommended method is, so you might need to do some research.

Reply:
I took this code straight from the micropython implementation. @cpswan should we be seeding random?

I'm not even sure if we need the IVNonce custom class (since it's from micropython). We can use the secrets module instead. 
 
`iv = secrets.token_bytes(16)`

LMK.

Reply:
@Xlin123 please open a standalone issue for this so it doesn't get overlooked.

Reply:
@cpswan I think this is what XC wanted but I'll check with him later. Fixed on this commit [6102da4](https://github.com/atsign-foundation/at_python/pull/31/commits/6102da458e32099d1ac4ae5d6eb958aba6464c20)